### PR TITLE
Replace Handlebars.SafeString in favor of Ember.String.htmlSafe

### PR DESCRIPTION
**Pull request by xomaczar**

### CODE REVIEW

File: addon/components/virtual-each/component.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const VirtualEachComponent = Component.extend(EventListenerMixin, DefaultAttrsMi
       const marginTop = escapeExpression(get(this, '_marginTop'));
       const height = escapeExpression(get(this, '_contentHeight'));
 
-      return new SafeString(`height: ${height}px; margin-top: ${marginTop}px;`);
+      return new htmlSafe(`height: ${height}px; margin-top: ${marginTop}px;`);
```
Comment (xomaczar) [inline addon/components/virtual-each/component.js]: Fixed

File: addon/components/virtual-each/component.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ const VirtualEachComponent = Component.extend(EventListenerMixin, DefaultAttrsMi
     get() {
       const height = escapeExpression(this.getAttr('height'));
 
-      return new SafeString(`height: ${height}px;`);
+      return new htmlSafe(`height: ${height}px;`);
```
Comment (xomaczar) [inline addon/components/virtual-each/component.js]: same as above

Comment (xomaczar) [inline addon/components/virtual-each/component.js]: can you restart travis

**Comment by xomaczar** [comment]
All green. Whenever you get a chance.

**Comment by jasonmit** [comment]
Thanks!

**Comment by jasonmit** [comment]
Versioned as 0.3.1

**Comment by xomaczar** [comment]
Thx

**Pull request merged**